docs/Configuring.md

```diff
@@ -506,27 +507,52 @@ When `invalidate-bad-cache-results=true` (default: `true`), Cromwell will invali
 
 Cromwell also accepts [Workflow Options](wf_options/Overview#call-caching-options) to override the cache read/write behavior.  
 
+#### Call cache strategy options
+
+* hash based options. These read the entire file. These strategies work with containers.
+    * `xxh64`. This uses the 64-bit implementation of the [xxHash](https://www.xxhash.com)
+             algorithm. This algorithm is optimized for file integrity hashing and provides a more than 10x speed improvement over
+             md5. 
+    * `md5`. The well-known md5sum algorithm
+* Path based options. These are based on filepath. Extremely lightweight, but only work with the `soft-link` file 
+caching strategy and can therefore never work with containers.
+    * `path` creates a md5 hash of the path.
+    * `path+modtime` creates a md5 hash of the path and its modification time.
+* Fingerprinting. This strategy works with containers.
+    * `fingerprint` tries to create a fingerprint for each file by taking its last modified time (milliseconds since
+       epoch in hexadecimal) + size (bytes in hexadecimal) + the xxh64 sum of the first 10 MB of the file. It is much
+       more lightweight than the hash based options while still unique enough that collisions are infeasible. This 
```

Comment:
```suggestion
       more lightweight than the hash based options while still unique enough that collisions are unlikely. This 
```

Reply:
Certainly collisions are feasible, but under normal conditions (and unless you're trying to force it to happen!) I'd say vanishingly unlikely.

Reply:
Agreed. This is why I took the first 10 MB, not the first 1 MB. Some bioinformatics files have *very* large headers. More than 10 million characters though? 
You can imagine that in BAM files the headers are the same, and even the first aligned reads are the same, so the first 10 MB are the same. But then the total file size should also be the same, and the jobs should have finished in exactly the same millisecond. It is indeed quite unlikely.
I changed the wording.